homeassistant/components/gree/bridge.py

```diff
@@ -68,29 +65,45 @@ async def push_state_update(self):
                 self.device.device_info,
             )
 
+    @property
+    def device_info(self):
+        """Return the gree device information."""
+        return self.device.device_info
```

Comment:
What's this for?

Reply:
Relic from an older refactor that removed wrappers in this bridge, this will be removed as it is unused.